src/ValidationError.js

```diff
@@ -12,8 +12,6 @@ class ValidationError extends Error {
 
     this.errors = errors;
 
-    Error.captureStackTrace(this, this.constructor);
-
     return this;
```

Comment:
Don't `return this` from a constructor.

Reply:
😱 I'm certainly a spastic with `this` from time to time 😛. Thx `err.stack` capturing works perfectly fine now